.ci/release/Jenkinsfile

```
@@ -13,6 +13,8 @@ pipeline {
     HOME = "${env.WORKSPACE}"
     JAVA_HOME = "${env.HUDSON_HOME}/.java/java10"
     PATH = "${env.JAVA_HOME}/bin:${env.PATH}"
+    SLACK_CHANNEL = '#apm-agent-java'
+    NOTIFY_TO = 'TBD'
```

what email?

---

@elastic/apm-agent-java , can you clarify what email do you want to receive it from? And whether it can be exposed here? Otherwise, I can ask the infra team to create a credentials entry with the email address if it helps

---

As suggested by @felixbarny, using the same thing as other agents do seems the best option:
We do not (yet) have email alias that I'm aware of though, +1 on creating one to make it consistent with others if required.

https://github.com/elastic/apm-agent-nodejs/blob/master/.ci/Jenkinsfile#L10

---

That's the generic one, so we can follow up if the generic one is not sufficient, then
```suggestion
    NOTIFY_TO = credentials('notify-to')
```

---

🐻

---

🤣 
I raised a PR to configure the credentials in the `internal-ci`, just waiting for the review and deployment.

---

Infra team suggested that we don't need to mask the email:


So I'll add the filter `apm-agent-java`

```suggestion
    NOTIFY_TO = 'build-apm+apm-agent-java@elastic.co'
```